pages/01_embedding_and_jsapi.md

```diff
@@ -7,11 +7,48 @@ The easiest method for embedding a Tableau view (dashboard or visualization) is
 ![Embed Code]({{ site.baseurl }}/img/embed_code.png)
 
 Once acquired, this code can be pasted into HTML.
-This method is useful for simple embedding, such as embedding into blogs or internal knowledge bases, but **only very simple embedding scenarios should use the embed code. Most deployments should instead use the JavaScript API.** It is not significantly more work to embed with the JavaScript API and doing so will gain you flexibility and power in your embedded deployment.
+This method is useful for simple embedding, such as embedding into blogs or internal knowledge bases, but **only very simple embedding scenarios should use the embed code. Most deployments should instead use the JavaScript API or the Embedding API v3.** It is not significantly more work to embed with the JavaScript API or the Embedding API v3, and doing so will gain you flexibility and power in your embedded deployment.
+
+## Embedding with the Embedding API v3
+
+The Embedding API v3 is the newest and easiest way to embed Tableau views, and provides you with options for enhancements, including improved security and integration for Connected Apps and external authorization servers (EAS). The Embedding API v3 is the new, modern replacement for the JavaScript API v2. 
+
+Available starting with Tableau 2021.4, you can use the Embedding API v3 to add a Tableau view as a web component in your HTML pages. Using the Tableau viz web component, you can complete your basic embed scenarios without having to write JavaScript code. The Embedding API v3 is currently available now for basic embedding. If you are looking to replace your embed code, or you are starting a new embedding project, you should consider using the Embedding API v3. 
+
```

Comment:
Replace this text:

>The Embedding API v3 is the newest and easiest way to embed Tableau views, and provides you with options for enhancements, including improved security and integration for Connected Apps and external authorization servers (EAS). The Embedding API v3 is the new, modern replacement for the JavaScript API v2. 
>
>Available starting with Tableau 2021.4, you can use the Embedding API v3 to add a Tableau view as a web component in your HTML pages. Using the Tableau viz web component, you can complete your basic embed scenarios without having to write JavaScript code. The Embedding API v3 is currently available now for basic embedding. If you are looking to replace your embed code, or you are starting a new embedding project, you should consider using the Embedding API v3.

The Embedding API v3 is the latest iteration of Tableau's embedding capabilities on the front-end of your application. It features a modern approach to initializing embedded content using [web components](https://www.webcomponents.org/introduction). It allows for improved security and integration capabilities via Connected Apps or via an External Authorization Server (EAS), which are both based on the JWT standard. When fully implemented later this year, the Embedding API v3 will support all the functionality of the JavaScript API v2, while providing future enhancements for the embedding functionality. If you are starting a new project, we recommend using this API.

Reply:
can we remove "When fully implemented later this year,"

Reply:
or rephrase it

Reply:
What about "In due time".. or "the Embedding API v3 will soon support all the functionality.."

Reply:
i am ok with the second option, thanks

Reply:
done